store/todos.js

```diff
@@ -1,5 +1,10 @@
 import { uuid } from 'uuidv4';
 export const state = () => ({
+  limit: 9,
```

Comment:
For values that doesn't change that much but may be used in different places, we tend to create a helper method, e.g: `constants.js` and export them from there. 

This helps us reduce a lot of repetitive task once we decide to change on of these value.

Reply:
Resolved